composer.json

```
@@ -9,17 +9,18 @@
         }
     ],
     "require": {
-        "php": "^5.5 || ^7.0",
+        "php": "^7.2",
         "nikic/php-parser": "^3.0 || ^4.0",
-        "symfony/finder": "^2.7 || ^3.0 || ^4.0",
-        "twig/twig": "^1.27 || ^2.0",
+        "symfony/finder": "^3.4 || ^4.3 || ^5.0",
```

I'd be ok with putting the 4.x branch on the LTS as well, as it's first patch version has been released earlier this week.

```suggestion
        "symfony/finder": "^3.4 || ^4.4 || ^5.0",
```

---

sf 4.3 is still officially supported until January 2020 (& July 2020 for security fixes) https://symfony.com/releases/4.3)
We could get rid of it but I don't think we should as we won't win anything to do it.